styles/layout/_styles.scss

```diff
@@ -59,3 +59,55 @@
 .right-justify {
     text-align: right;
 }
+
+.center {
+    margin: 0 auto;
+    text-align: center;
+}
+
+.inline-block-component {
+    display: inline-block;
+    width: 100%;
+}
+
+.copyright {
+    color: $color-base-0;
+    display: block;
+    font-size: $font-size-base;
+    margin: 0;
+    padding-top: $copyright-padding-top;
+    text-align: center;
+    width: 100%;
+}
+
+.daptiv-container-bg {
+    background-color: $color-primary-dark;
+    background-image: radial-gradient(ellipse farthest-side at center top, $color-primary-medium 0%, $color-primary-dark 100%);
+    background-position: top center;
+    background-repeat: no-repeat;
+    background-size: $daptiv-container-bg-bg-size;
+    height: $daptiv-container-bg-height;
+    min-height: $daptiv-container-bg-min-height;
+    width: $daptiv-container-bg-width;
+}
+
+.footer {
+    background-image: radial-gradient(50% 99%, $color-primary-dark 0%, $color-primary-dark-shadow 100%);
+    box-shadow: inset 0 3px 3px 0 $color-primary-dark-shadow;
+    height: $footer-height;
+    width: 100%;
+}
+
+.static-header {
+    padding-bottom: $static-header-padding-bottom;
+    padding-top: $static-header-padding-top;
+    width: $static-header-width;
+}
+
+.separator {
```

Comment:
Okay, I'll move inside. My idea behind this was to be able to reuse in other places